vestido/vestido-storefront/modules/Checkout/CheckoutView.tsx

```diff
@@ -79,15 +86,32 @@ const CheckoutView: React.FC = () => {
 
   const { trigger: createOrderTrigger } = useCreateOrder();
 
-  const totalPrice =
-    cartItems?.data.reduce((total, item) => {
-      return total + item.qty * item.item.price;
-    }, 0) ?? 0;
+  const totalPrice = buyNowItem
+    ? (checkoutItems?.reduce((total, item) => {
+        return total + item.item.price;
+      }, 0) ?? 0)
+    : (cartItems?.reduce((total, item) => {
+        return total + item.qty * item.item.price;
+      }, 0) ?? 0);
 
   const handleSubmit = async (data: CreateOrderForm) => {
+    const orderData = buyNowItem
+      ? {
+          addressId: data.addressId,
+          orderItems: [
+            {
+              itemId: buyNowItemId ?? '',
+              qty: 1,
+              price: buyNowItem.price,
+              variantId: buyNowVariantId ?? '',
+            },
+          ],
+          paymentType: data.paymentType,
+        }
+      : data;
```

Comment:
Form already has values from `checkoutItems` so this is unnecessary?

Reply:
` useEffect(() => {
    if (cartItems?.length) {
      form.setValue(
        'orderItems',
        cartItems?.map((cartItem) => ({
          itemId: cartItem.itemId,
          price: cartItem.item.price,
          qty: cartItem.qty,
          variantId: cartItem.variantId ?? null,
        })),
      );
    }
  }, [cartItems, form]);`
  
  this should be changed to checkoutItems ??

Reply:
but then what about qty or variantID ?

Reply:
![image](https://github.com/user-attachments/assets/685bd723-303d-4a43-8682-4cfc0fb93062)
first, i should add a constVariantId???

Reply:
this is done